data/pokedex.ts

```diff
@@ -12077,6 +12358,23 @@ export const Pokedex: {[speciesid: string]: SpeciesData} = {
 		prevo: "Sliggoo",
 		evoLevel: 50,
 		eggGroups: ["Dragon"],
+		otherFormes: ["Goodra-Hisui"],
+		formeOrder: ["Goodra", "Goodra-Hisui"],
+	},
+	goodrahisui: {
+		num: 706,
+		name: "Goodra-Hisui",
+		baseSpecies: "Goodra",
+		forme: "Hisui",
+		types: ["Dragon"],
```

Comment:
```suggestion
		types: ["Steel", "Dragon"],
```

Reply:
I got a bit copypaste happy towards the end… oops